webapp/package.json

```diff
@@ -74,8 +74,8 @@
     "webpack-cli": "5.0.1"
   },
   "dependencies": {
-    "@mattermost/webapp": "github:mattermost/mattermost-webapp",
     "core-js": "3.22.8",
+    "mattermost-redux": "5.33.1",
```

Comment:
Uff, that is quite an old version. I vaguely remember plans to publish the content from `mattermost-webapp/pacckages/mattermost-redux` as it's own npm package. Did these plans lead anywhere?

Reply:
My original plan was to replace it with something better. I can't remember if I reconsidered that at some point, but I'm wanting a proper `@mattermost/plugin-api` package currently.

I don't think the more common uses of `mattermost-redux` have changed much since 5.33.1, so I feel like this version is probably fine

Reply:
@hanzei Thoughts on this PR with the above in mind?

Reply:
Do we know if what is in `mattermost-redux` is "enough" for most plugins? E.g. do GitHub, JIRA and MS Teams compile with only `mattermost-redux`?

Reply:
@hanzei Indeed they are all using `mattermost-redux` as their dependency:

https://github.com/mattermost/mattermost-plugin-github/blob/9469235a390c96f675d2cafccd7f8f267f4bff5c/webapp/package.json#L70

https://github.com/mattermost/mattermost-plugin-msteams-meetings/blob/d895e1a5326eae6c2f4b80047f5f333f34056560/webapp/package.json#L22

https://github.com/mattermost/mattermost-plugin-jira/blob/b4b11e93c652aed2f99f73c725e243b46f4986c3/webapp/package.json#L68